src/main/java/org/polyfill/services/FinancialTimesPolyfillLoaderService.java

```diff
@@ -0,0 +1,120 @@
+package org.polyfill.services;
+
+import org.polyfill.components.Polyfill;
+import org.polyfill.interfaces.ConfigLoaderService;
+import org.polyfill.interfaces.PolyfillLoaderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by smo on 2/25/17.
+ * Service to load and construct Financial Times' polyfills from file system.
+ * Financial Times' polyfill folder structure:
+ * polyfillName
+ * - meta.json
+ * - min.js
+ * - raw.js
+ */
+@Primary
+@Service("directory based")
+@SuppressWarnings("unchecked")
+public class FinancialTimesPolyfillLoaderService implements PolyfillLoaderService {
```

Comment:
Renamed from DirectoryBasedPolyfillLoaderService. This is specific to Financial Times' structure and will be the only file to modify if the file structure is different.